index.d.ts

```diff
@@ -19,6 +19,8 @@ declare module "react-dropdown" {
     menuClassName?: string;
     arrowClassName?: string;
     disabled?: boolean;
+    arrowClosed?: JSX.Element,
+    arrowOpen?: JSX.Element,
```

Comment:
Should this be a `ReactNode` or `JSX.Element`? I'm not so sure and just curious.

Reply:
ReactNode is pointful to use since both ```ReactNode``` and ```JSX.Element``` presetn DOM-instance (still in different ways). Also I was suggested to use ```ReactElement``` but this type is generic and requiers additional type checking from end-users. As for now, it's okay to type these as ```React.ReactNode```. What do you think?

Reply:
That sounds good to me. Thanks for the explanation!